kimchi/wasm/src/plonk_proof.rs

```diff
@@ -690,7 +689,7 @@ macro_rules! impl_proof {
                     rot_selector: None,
                     lookup_aggregation: None,
                     lookup_table: None,
-                    lookup_sorted: array::from_fn(|_| None),
+                    lookup_sorted: array_init(|_| None),
```

Comment:
Why? Is it somehow more efficient to use `array_init`? If not, I think it's better to used the standard library.

Reply:
I would tend to exactly do the contrary, i.e., change all calls to `array_init` to use `from_fn` to remove the dependency on a given `crate`

Reply:
Mmh, I don't have any real reason except than staying consistent. I assumed there was a good reason before to use `array_init` everywhere.